apps/wing/src/commands/test.ts

```diff
@@ -29,6 +30,25 @@ export interface TestOptions extends CompileOptions {
 }
 
 export async function test(entrypoints: string[], options: TestOptions): Promise<number> {
+  // Initialize a Set to store unique file names
+  let expandedEntrypointsSet: Set<string> = new Set();
+
+  // Expand each entrypoint and add file names to the Set
+  for (const entrypoint of entrypoints) {
+    const expandedEntrypoint = globSync(entrypoint);
```

Comment:
Use the async glob instead.
Also I think glob can take an array, so there may be no need to deal with the Set above (it probably deals with duplicates)